packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -122,6 +123,12 @@ class PerformanceMonitoring {
           transactionService.removeTask(task.id)
         }
       }
+
+      if (event === INVOKE && task.source === HISTORY_PUSHSTATE) {
+        transactionService.startTransaction(task.data.title, 'route-change', {
```

Comment:
The title can be empty and not sure if it's set all the time. we have to handle this case and I am not sure if it's set all the time during popstate events.

Reply:
The title is not used in any of the frameworks I've seen. But we don't have to handle it here since the transaction title can be undefined and if it is then it would have 'unknown'.

Reply:
Wont it be confusing then? If we are relying on title for the transaction name and multiple paths might lead to `Unknown` because of title being empty on different routes?